gunicorn/workers/base.py

```diff
@@ -132,6 +130,9 @@ def changed(fname):
             reloader_cls = reloader_engines[self.cfg.reload_engine]
             self.reloader = reloader_cls(extra_files=self.cfg.reload_extra_files,
                                          callback=changed)
+
+        self.load_wsgi()
+        if self.reloader:
```

Comment:
Why do we need this branch? Can't we just simply revert https://github.com/benoitc/gunicorn/commit/3054b184dea1b07581293c203a1c546c7ea3ebca?

Reply:
That commit was meant as a fix for #2133 (inotify reloader does not pick up files), but also introduced the regression in  #2244 (syntax errors crash the process on reload). So a simple revert would fix the regression but also re-break the inotify reloader unfortunately.

Doing things in this particular order (load_wsgi, then start the reloader) fixes the original problem, but without the regression.